core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -542,6 +542,25 @@ struct HIPParallelLaunchKernelInvoker<DriverType, LaunchBounds,
     HIPInternal::constantMemReusable[hip_device].release(
         std::move(lock), hip_instance->m_stream);
   }
+
+  inline static void create_parallel_launch_graph_node(
```

Comment:
```suggestion
  static void create_parallel_launch_graph_node(
```
(consistency within that source file)

Reply:
OK, more consistent indeed.